dags/main_summary_backfill_mar_2018.py

```diff
@@ -65,13 +64,6 @@ def main_summary_subdag_factory(parent_dag, task_id, day):
     parent_job_flow_id = ("{{{{ task_instance.xcom_pull('setup_backfill_cluster', "
                           "key='return_value', dag_id={}) }}}}".format(parent_dag.dag_id))
 
-    # Try to alleviate throttling issues by introducing some slight jitter on each of the days
```

Comment:
Is removing this related to the limited number of celery workers?

Reply:
No, we just don't need to introduce artificial jitter anymore since we're not submitting the steps all at once